### DISCUSSION TITLE
friday-updates/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#9] [Announcements]
# friday-updates/

Team Updates
A number of members of my team at Valve don’t blog, and I like to bring visibility to their work when I can. Here’s a quick roundup of what some of them have been doing, in no particular order.

https://www.supergoodcode.com/friday-updates/

**Comment by norablackcat** [comment]
Shhhh caption code deletion makes it too obvious

**Comment by qwertychouskie** [comment]
It's been over a week with no new posts!  Don't you realize that your job, first and foremost, is to write entertaining blog posts?

**Reply by qwertychouskie** [reply]
Halloooooooooooooooooooooooooo

**Comment by boberfly** [comment]
Amazing work! With sparse-binding for lavapipe being the missing piece, does that suggest raytracing works already on it? This could be huge for VFX/CG offline rendering of UnrealEngine 5 on renderfarms where it's not common yet (or economical) to run on GPUs - or scale up with more available ram.

**Reply by jams3223** [reply]
Can you explain to me how will they use Lavapipe with VKD3C-Proton, as i don't understand what benefit it could bring in this work.

**Comment by jams3223** [comment]
What about Timur ?
